### DISCUSSION TITLE
CITATION.cff under .github

### DISCUSSION THREAD

**Discussion by sorawee** [community/community#41061] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

GitHub supports the `CITATION.cff` metadata (https://github.blog/2021-08-19-enhanced-support-citations-github/). However, it requires that the file must be at the project root. This is undesirable for projects that wish to keep the project root uncluttered.

We would prefer if GitHub could also look for `CITATION.cff` under the `.github` directory. The directory is already used for many other things, like workflows, issue template, etc. This would allow us to keep the project root tidy while benefiting from the feature.

Thanks!

**Comment by jfullstackdev** [comment]
what would being tidy results to ? is there any other practical benefit other than being tidy in the root ?

**Reply by sorawee** [reply]
README is shown after file listing. Having many files means anyone who visits the page must scroll down farther than needed.

It's also about organization and consistency. Most GitHub-specific features, like issue templates and workflows, consult the `.github` directory.

**Reply by jfullstackdev** [reply]
I'm not very sure of your situation but any file can simply be search at `Go to file` feature because, like in PHP Laravel Development, it's sometimes very hard to find a specific file, whether a view or a controller or a migration file, but we can't change the folder structure of PHP Laravel, so the best solution is to simply use the `Go to file` feature of GitHub and it's problem solved

**Comment by sorawee** [comment]
It looks like GitHub can already support looking for the citation file in non-root locations. From https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#other-citation-files:

```
# Note these are case-insensitive and must be in the root of the repository
CITATION
CITATIONS
CITATION.bib
CITATIONS.bib
CITATION.md
CITATIONS.md

# CITATION files for R packages are typically found at inst/CITATION
inst/CITATION
```

This request simply asks that `.github/CITATION.cff` is added to the above list.

**Reply by vilkinsons** [reply]
+1 for this. On the basis that alternative citation locations are already supported, for consistency with the allowed `.github` placement location for `CODEOWNERS`, `SECURITY.md`, `CONTRIBUTING.md`, `CODE_OF_CONDUCT.md` and other repo-level files... it feels like this should be supported.

**Reply by spdegabrielle** [reply]
+1 for this - here is an example in a prominent open source project

https://github.com/racket/racket/issues/4140#issuecomment-1022176647
